compiler/passes/normalize.cpp

```diff
@@ -2401,6 +2401,17 @@ static void normVarTypeWithInit(DefExpr* defExpr) {
     } else {
       Expr*     arg     = toCallExpr(initExpr)->get(1)->remove();
       CallExpr* argExpr = toCallExpr(arg);
+      SymExpr*  modToken = NULL;
+      SymExpr*  modValue = NULL;
+
+      if (argExpr->numActuals() >= 2) {
+        if (SymExpr* se = toSymExpr(argExpr->get(1))) {
+          if (se->symbol() == gModuleToken) {
+            modToken = toSymExpr(argExpr->get(1)->remove());
```

Comment:
I would be tempted to just say `modToken = se->remove();`, since you already have it.  I'd also leave a comment on the modValue assignment reinforcing that the result of get(1) there is different than the value returned earlier (or just swap the order of the assignments so that you get(2) and then remove the first argument).